### PR TITLE
Updated Hashset Hashmap and Arrraylist

### PR DESCRIPTION
**Pull request by kshitijmishra27**
Hashmap Hashset and arraylist are important collections in Java . So i think general information about Hashpmap , hashset and array list should be provided. So i added the basic information about these 

Hacktoberfest ❤

### CODE REVIEW

File: java.md [hunk 1]
```diff
@@ -238,6 +238,77 @@ do{
 |Deque| Deque(Double Ended Queue) is used to add or remove elements from both the ends of the Queue(both head and tail)|
 |Map| Map contains key-values pairs which don't have any duplicates. Map is implemented in HashMap, TreeMap etc.|
 
+## ArrayList
+
+The ArrayList class is a resizable array, which can be found in the java.util package.
+
+### Example
+
+```c
```
Comment (onecompiler-ops) [inline java.md:247]: use java code formatting instead `c`

File: java.md [hunk 2]
```diff
@@ -238,6 +238,77 @@ do{
...
+
+```
+
+## HashSet
+
+A HashSet is a collection of items where every item is unique, and it is found in the java.util package:
+ 
+ ###  Example
+ 
+ ```c
```
Comment (onecompiler-ops) [inline java.md:270]: use java code formatting instead `c`

File: java.md [hunk 3]
```diff
@@ -238,6 +238,77 @@ do{
...
+  }
+}
+ 
+ ```
+ 
+ ## HashMap
+ 
+  A HashMap however, store items in "key/value" pairs, and you can access them by an index of another type (e.g. a String).
+  
+  ```c
```
Comment (onecompiler-ops) [inline java.md:292]: use java code formatting instead c

**Review by onecompiler-ops — CHANGES_REQUESTED** [review]

File: java.md (hunk 1 above)
Comment (kshitijmishra27) [reply java.md]: updated the code

**Review by kshitijmishra27 — COMMENTED** [review]
done changes

**Review by onecompiler-ops — APPROVED** [review]

**Comment by onecompiler-ops** [comment]
Thank you for the updates, merging it.

**Comment by kshitijmishra27** [comment]
@onecompiler-ops  done the changes😄

**Pull request merged**